### PR TITLE
Helm: support select value files

### PR DESCRIPTION
**Pull request by wangyikewxgm**
Signed-off-by: 楚岳 <wangyike.wyk@alibaba-inc.com>

### CODE REVIEW

File: docs/tutorials/helm-multi-cluster.md [hunk 1]
```diff
@@ -317,6 +317,82 @@ If you're using the `velaux` UI console, you can get even more information with
...
+If you choose ```Cluster: local | Namespace: prod | Kind: HelmRelease | Name: hello-kubernetes``` you will see:
+
+![image](../resources/helm-files-prod.jpg)
+
+
+
+
+
+
+
```
Comment (wonderflow) [inline docs/tutorials/helm-multi-cluster.md]: please trim the space line

File: docs/tutorials/helm-multi-cluster.md [hunk 2]
```diff
@@ -317,6 +317,82 @@ If you're using the `velaux` UI console, you can get even more information with
 
 ![resource-detail](../resources/helm-topology.jpg)
 
+## Specify different value file for different environment
+
+You can choose different value file present in a helm chart for different environment. eg:
+
+First guarantee your local cluster have two namespaces "test" and "prod" which represent test and prod environment.
```
Comment (wonderflow) [inline docs/tutorials/helm-multi-cluster.md]:
```suggestion
Please make sure your local cluster have two namespaces "test" and "prod" which represent two environments in our example.
```

File: docs/tutorials/helm.md [hunk 3]
```diff
@@ -161,6 +161,30 @@ You can hover your mouse on it to check the health status, by clicking the icon
 
 At this point, Helm Chart in KubeVela is no stranger to you, go ahead and try more!
 
+## Specify different value file
+
+You can change the default values file `values.yaml` with another value file present inside the Helm chart by set the `valuesFiles` field. eg:
```
Comment (wonderflow) [inline docs/tutorials/helm.md:166]: you should list directory of the chart to make sure users understood they should have the `values-production.yaml` inside the chart：
```
$ tree ./hello-kubernetes-chart
...
values.yaml
values-productions.yaml
...
```

File: versioned_docs/version-v1.4/tutorials/helm-multi-cluster.md [hunk 4]
```diff
@@ -317,6 +317,82 @@ If you're using the `velaux` UI console, you can get even more information with
...
+        clusters: ["local"]
+        namespace: "prod"
+    - name: override-prod
+      type: override
+      properties:
+        components:
+          - name: hello-kubernetes
+            properties:
+              valuesFiles:
+                - "values-production.yaml"
```
Comment (barnettZQG) [inline versioned_docs/version-v1.4/tutorials/helm-multi-cluster.md:383]: This use case needs to be explained:

```yaml
- name: override-test
      type: override
      properties:
        components:
          - name: hello-kubernetes
            properties:
              valuesFiles:
                - "values.yaml"
                - "values-test.yaml"
- name: override-prod
      type: override
      properties:
        components:
          - name: hello-kubernetes
            properties:
              valuesFiles:
                - "values.yaml"
                - "values-production.yaml"
```

Comment (wangyikewxgm) [reply versioned_docs/version-v1.4/tutorials/helm-multi-cluster.md:383]: This doc has explained.

Comment (wonderflow) [reply versioned_docs/version-v1.4/tutorials/helm-multi-cluster.md:383]: Yes, we should explain what will happen if we write more than 1 files.

**Review by wonderflow — APPROVED** [review]
good job

**Pull request merged**